content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -64,15 +64,14 @@ type and enforces the appropriate maximum number of volumes for the node. For ex
 
 * On
 <a href="https://cloud.google.com/compute/">Google Compute Engine</a>,
-up to 127 volumes can be attached to a node, [depending on the node
+up to 127 volumes can be attached to a Node, [depending on the Node
 type](https://cloud.google.com/compute/docs/disks/#pdnumberlimits).
 
-* For Amazon EBS disks on M5,C5,R5,T3 and Z1D instance types, Kubernetes allows only 25
-volumes to be attached to a Node. For other instance types on
+* On
 <a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>,
```

Comment:
Rather than repeat the number, we should direct people to read the driver or cloud vendor documentation to find the actual limit.

Reply:
It's OK to hyperlink to the specific page that answers that question.